couchbase/search/v1/search.proto

```diff
@@ -274,6 +286,9 @@ message SearchQueryRequest {
   bool include_locations = 13;
   map<string, Facet> facets = 14;
 
+  repeated KnnQuery knn = 17;
```

Comment:
In SDKland we've purposely avoided the wording of knn as fts could decide to use another algorithm at some point. Do we have a plan on how we'd handle that in PS?

Reply:
I think in this case, the vector search query _must_ be a KNN query, since it has properties that are specific to KNN (like the K value), I suspect other forms of vector search would appear under a unique query property name.